stack-lts-18.0.yaml

```diff
@@ -0,0 +1,30 @@
+resolver: lts-18.0
+
+packages:
+- aws-xray-client
+- aws-xray-client-wai
+- aws-xray-client-persistent
+
+ghc-options:
```

Comment:
I might not worry about this. Let our warning-preferences be `stack.yaml`-only to just cut down on the amount of copy/paste.